app/components/dashboard/approvals_component.html.erb

```diff
@@ -15,7 +15,7 @@
     <% approvals.each do |submission| %>
       <tr>
         <%= render Works::LinkToShowComponent.new(work_version: submission) %>
-        <td><%= submission.work.depositor.sunetid %></td>
+        <td><%= submission.work.owner.sunetid %></td>
```

Comment:
That's true - let's chat at standup and see what we want to do (maybe change the label to match).  I think this may be one of the few/only spots where we have some ambiguity in display like this.  To me it makes sense to show whomever the current responsible person for the work is (owner), instead of the original depositor (which of course will most often be the same anyway).

Reply:
Per standup discussion, switched label to say "Owner".